# do not build tests. they require to compile the gmock-1.6.0.zip and gtest-1.6.0.zip files.
Index: synergy-1.8.4/src/CMakeLists.txt
===================================================================
--- synergy-1.8.4.orig/src/CMakeLists.txt	2016-10-11 17:59:58.465014900 +0200
+++ synergy-1.8.4/src/CMakeLists.txt	2016-10-11 17:59:58.461014822 +0200
@@ -50,6 +50,4 @@
 add_subdirectory(cmd)
 add_subdirectory(micro)
 
-if (NOT ${CMAKE_SYSTEM_NAME} MATCHES "IRIX")
-	add_subdirectory(test)
-endif()
+
